src/main/groovy/org/sharegov/mdcgis/AddressController.groovy

```diff
@@ -125,6 +161,31 @@ class AddressController {
 		json.call(answer)
 	}
 
+	/**
+	 * Client and get Municipality ID
```

Comment:
can you explain what this method does ? "Client an get Municipality ID" does not sound right